pkg/permissions/generate.go

```diff
@@ -19,6 +20,80 @@ type AgentRBACManifests struct {
 	RoleBindings []rbac.RoleBinding
 }
 
+//*******************************
+func AgentCLR(clusterRole []rbac.ClusterRole) string {
+	var got []string
+	for _, clr := range clusterRole {
+		var aG, rC, vB string
+		for _, rule := range clr.Rules {
+			aG = strings.Join(rule.APIGroups, "\", \"")
+			rC = strings.Join(rule.Resources, "\", \"")
+			vB = strings.Join(rule.Verbs, "\", \"")
+		}
+
+		got = append(got, fmt.Sprintf(
+			`apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+	name: jetstack-secure-agent-pods-reader
```

Comment:
There are too many whitespaces before `name: jetstack-secure-agent-pods-reader`

Reply:
The name shouldn't be `jetstack-secure-agent-pods-reader` but should be the name present in `clr` like `clr.ObjectMeta.Name`

Reply:
I did a tab here, which seems to be 4 spaces, and the expected and other parts are all using tab as well, so I thought it will be easier to line up.